lib/rubocop/cop/style/if_with_semicolon.rb

```diff
@@ -33,10 +34,39 @@ def on_normal_if_unless(node)
         private
 
         def correct_to_ternary(node)
```

Comment:
This method name doesn't make sense anymore if sometimes it's not a ternary.

Reply:
Sounds good!